changelogs/fragments/docker_network-requirements.yaml

```diff
@@ -1,6 +1,8 @@
+---
```

Comment:
This file should keep it's name as well. Only move the version bumps to the new file.

Reply:
ah, reverting...

Reply:
Looks good now :) Thanks!